fedimint-core/src/config.rs

```diff
@@ -138,6 +138,13 @@ pub struct ConfigResponse {
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, Hash, PartialEq, Encodable)]
 pub struct FederationId(pub threshold_crypto::PublicKey);
 
+impl Display for FederationId {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        let hex = self.0.to_bytes().to_hex();
+        write!(f, "{hex}")
+    }
```

Comment:
we should be able to use  [`format_hex`](https://docs.rs/bitcoin_hashes/latest/bitcoin_hashes/hex/fn.format_hex.html) here.
it's more efficient than going through `ToHex`.  [we also do this here](https://github.com/fedimint/fedimint/blob/master/fedimint-core/src/fmt_utils.rs#L22-L25)

Reply:
TIL about `format_hex`. Updated to use it.